metagraph/src/graph/representation/succinct/boss_chunk_construct.cpp

```diff
@@ -703,7 +819,7 @@ void recover_dummy_nodes(const KmerCollector &kmer_collector,
                     return kmer::transform<KMER>(reinterpret_cast<const KMER_REAL &>(v), k + 1) + kmer_delta;
                 }
             },
-            real_split_by_W, true
+            real_split_by_W, false /* remove sources */
```

Comment:
looks like a variable name commented out
```suggestion
            real_split_by_W, false // remove sources
```